libmamba/src/core/url.cpp

```diff
@@ -155,6 +155,7 @@ namespace mamba
         }
 
         // mimicking conda's behavior by special handling repodata.json
+        // todo support .zst
```

Comment:
I think we just keep the `repodata.json` as filename because that is the actual content of the file.

For `current_repodata.json` it's a different hash but the contents are also different :)